voxel_mesher/main.cpp

```diff
@@ -118,94 +118,94 @@ class CubeGeometryBuilder
 		if (faces[NorthFace])
 		{
 			SetNormal( 0, 0, -1 );
-			SetSetUV(uvRect.x, uvRect.y);
-			PushVertex(position);
+            SetSetUV(uvRect.width, uvRect.height);
+            PushVertex(position);
 
-			SetSetUV(uvRect.width, uvRect.height);
-			PushVertex(position, 1, 1, 0);
+            SetSetUV(uvRect.x, uvRect.y);
+            PushVertex(position, 1, 1, 0);
 
-			SetSetUV(uvRect.width, uvRect.y);
-			PushVertex(position, 1, 0, 0);
+            SetSetUV(uvRect.x, uvRect.height);
+            PushVertex(position, 1, 0, 0);
 
-			SetSetUV(uvRect.x, uvRect.y);
-			PushVertex(position);
+            SetSetUV(uvRect.width, uvRect.height);
+            PushVertex(position);
 
-			SetSetUV(uvRect.x, uvRect.height);
-			PushVertex(position, 0, 1, 0);
+            SetSetUV(uvRect.width, uvRect.y);
+            PushVertex(position, 0, 1, 0);
 
-			SetSetUV(uvRect.width, uvRect.height);
-			PushVertex(position, 1, 1, 0);
+            SetSetUV(uvRect.x, uvRect.y);
+            PushVertex(position, 1, 1, 0);
```

Comment:
The whitespace here does not match the existing code, perhaps your tabs as spaces are not set correctly?

Reply:
That's strange.  I think all my editors are set to 4 space indentation which I think is correct looking at Raylib guidelines.  I'll review this and resubmit.  Thanks.